[2023-04-12T04:35:14.488+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_extraction_dag.task_athlete_stats__1 scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2023-04-12T04:35:14.506+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_extraction_dag.task_athlete_stats__1 scheduled__2023-04-11T00:00:00+00:00 [queued]>
[2023-04-12T04:35:14.508+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T04:35:14.510+0000] {taskinstance.py:1289} INFO - Starting attempt 2 of 2
[2023-04-12T04:35:14.513+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-12T04:35:14.540+0000] {taskinstance.py:1309} INFO - Executing <Task(_PythonDecoratedOperator): task_athlete_stats__1> on 2023-04-11 00:00:00+00:00
[2023-04-12T04:35:14.600+0000] {standard_task_runner.py:55} INFO - Started process 643 to run task
[2023-04-12T04:35:14.605+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_extraction_dag', 'task_athlete_stats__1', 'scheduled__2023-04-11T00:00:00+00:00', '--job-id', '266', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion.py', '--cfg-path', '/tmp/tmpz7x1n4hx']
[2023-04-12T04:35:14.610+0000] {standard_task_runner.py:83} INFO - Job 266: Subtask task_athlete_stats__1
[2023-04-12T04:35:14.745+0000] {task_command.py:389} INFO - Running <TaskInstance: data_extraction_dag.task_athlete_stats__1 scheduled__2023-04-11T00:00:00+00:00 [running]> on host 392d142fd52f
[2023-04-12T04:35:14.964+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_extraction_dag
AIRFLOW_CTX_TASK_ID=task_athlete_stats__1
AIRFLOW_CTX_EXECUTION_DATE=2023-04-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-11T00:00:00+00:00
[2023-04-12T04:35:14.991+0000] {logging_mixin.py:137} INFO - Fetching for athlete stats started
[2023-04-12T04:37:10.654+0000] {logging_mixin.py:137} INFO - Fetching for athlete stats ended
[2023-04-12T04:37:10.655+0000] {python.py:177} INFO - Done. Returned value was:                          name  ... rankDisplayValue
0               fumblesForced  ...              NaN
1            fumblesRecovered  ...              NaN
2       fumblesRecoveredYards  ...              NaN
3           fumblesTouchdowns  ...              NaN
4                 gamesPlayed  ...              NaN
...                       ...  ...              ...
170178     totalTwoPointConvs  ...              NaN
170179      twoPointPassConvs  ...              NaN
170180       twoPointRecConvs  ...              NaN
170181      twoPointRushConvs  ...              NaN
170182      onePtSafetiesMade  ...              NaN

[170183 rows x 13 columns]
[2023-04-12T04:37:10.702+0000] {xcom.py:629} ERROR - Object of type DataFrame is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2023-04-12T04:37:10.705+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 2298, in xcom_push
    XCom.set(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/xcom.py", line 234, in set
    value = cls.serialize_value(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/xcom.py", line 627, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.9/json/__init__.py", line 234, in dumps
    return cls(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/json.py", line 176, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.9/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.9/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/json.py", line 170, in default
    return super().default(o)
  File "/usr/local/lib/python3.9/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable
[2023-04-12T04:37:10.717+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=data_extraction_dag, task_id=task_athlete_stats__1, execution_date=20230411T000000, start_date=20230412T043514, end_date=20230412T043710
[2023-04-12T04:37:10.744+0000] {standard_task_runner.py:100} ERROR - Failed to execute job 266 for task task_athlete_stats__1 (Object of type DataFrame is not JSON serializable; 643)
[2023-04-12T04:37:10.783+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-12T04:37:10.820+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
